sys/include/net/nanocoap.h

```diff
@@ -42,69 +44,7 @@ extern "C" {
 #endif
 
 /**
- * @brief   CoAP port to use
- */
-#define COAP_PORT               (5683)
-
-/**
- * @name    Nanocoap specific maximum values
- * @{
- */
-#define NANOCOAP_NOPTS_MAX      (16)
-#define NANOCOAP_URI_MAX        (64)
-/** @} */
-
-#ifdef MODULE_GCOAP
-#define NANOCOAP_URL_MAX        NANOCOAP_URI_MAX
```

Comment:
This *could* be moved within the group above, but this can also be done in a follow-up.